code/obj/item/organs/head.dm

```diff
@@ -333,7 +336,10 @@
 			else
 				boutput(user, "<span class='alert'>[src.name] is already wearing [src.glasses.name]!</span>")
 			return
-
+		if (istype(W, /obj/item/cloth))
+			playsound(src, "sound/items/towel.ogg", 25, 1)
+			user.visible_message("<span class='notice'>[user] [pick("buffs", "shines", "cleans", "wipes", "polishes")] [src] with [W].</span>")
+			return
```

Comment:
don't see how this is cleaning off blood stains and such

Reply:
It was just for flavor but I can make it clean blood stains and such too.

Reply:
i thought you meant by
> Cloths can be used on the head object to clean it
that it actually cleaned it lol